### PR TITLE
Summarise using by columns

### PR DESCRIPTION
**Pull request by cristineguadelupe**
Screenshot 2023-04-04 at 15 53 45

### CODE REVIEW

File: lib/assets/data_transform_cell/main.js [hunk 1]
```diff
@@ -543,26 +543,26 @@ export async function init(ctx, payload) {
...
+                            v-model="operation.query"
+                            :options="summariseUsingOptions"
+                            :index="index"
+                            :disabled="noDataFrame"
+                          />
                           <BaseMultiTagSelect
                             @change.self="addInnerValue($event)"
                             operation_type="summarise"
-                            label="Summarise by"
+                            label="By columns"
```
Comment (josevalim) [inline lib/assets/data_transform_cell/main.js:558]: I am not sure if this is correct. We may need "Of columns". I will check with a native english speaker.

Comment (cristineguadelupe) [reply lib/assets/data_transform_cell/main.js:558]: How about just "Columns"?

Comment (josevalim) [reply lib/assets/data_transform_cell/main.js:558]: I am thinking either "Columns" or "Of columns". I will let you know. :)

Comment (josevalim) [reply lib/assets/data_transform_cell/main.js:558]:
```suggestion
                            label="Of columns"
```

**Review by josevalim — APPROVED** [review]

**Pull request merged**